src/main/java/OperatorTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import util.Operator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OperatorTest {
+    private Operator operator;
+    @BeforeEach
+    void setUp() {
+        operator = new Operator();
+    }
+    @DisplayName("+")
+    @ParameterizedTest
+    @CsvSource("'1','2'")
+    void  plusTest(double a, double b) {
+        assertThat(3).isEqualTo(operator.plus(a,b));
```

Comment:
operator.plus(a,b) 반환 값이 더블이라면, 아래처럼 해야하지 않을까요?
```suggestion
        assertThat(3.0).isEqualTo(operator.plus(a,b));
```